.github/workflows/0-everything.yml

```diff
@@ -134,6 +134,7 @@ jobs:
         policyType:
           - DeployBuiltInPolicy
           - DeployCustomPolicy
+      fail-fast: false
```

Comment:
Suggest moving the `fail-fast: false` specifier to the line directly after 133 (the opening `matrix:` statement) for order consistency with other uses.